src/main/resources/swagger/api-docs.yaml

```diff
@@ -356,6 +356,138 @@ paths:
+      responses:
+        200:
+          description: the indicated method namespace ACL permissions once the posted changes have been applied
+          schema:
+            $ref: '#/definitions/MethodConfigACL'
+        400:
+          description: Invalid content format
+        403:
+          description: User does not have correct permissions to manage namespace permissions or the indicated namespace was not found
+        500:
+          description: Internal Server Error
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+            - https://www.googleapis.com/auth/devstorage.full_control
+  /api/configurations/{namespace}/permissions:
+    get:
+      tags:
+        - Method Repository
+      operationId: getConfigNamespaceACL
+      summary: get ACL permissions on a Method Repository Configuration Namespace
+      produces:
+        - application/json
+      parameters:
+        - name: namespace
+          description: Method Configuration namespace
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
+          description: List of the indicated configuration namespace ACL permissions
+          schema:
+            $ref: '#/definitions/MethodConfigACL'
+        403:
+          description: user does not have correct permissions to manage namespace permissions or the indicated namespace was not found
+        500:
+          description: Internal Server Error
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+            - https://www.googleapis.com/auth/devstorage.full_control
+    post:
+      tags:
+        - Method Repository
+      operationId: setConfigNamespaceACL
+      summary: set ACL permissions on a Method Repository Configuration Namespace
+      produces:
+        - application/json
+      parameters:
+        - name: namespace
+          description: Method Configuration namespace
+          required: true
+          type: string
+          in: path
+        - name: payload
+          description: the ACLs to upsert
+          required: true
+          in: body
+          schema:
+            $ref: '#/definitions/MethodConfigACL'
+      responses:
+        200:
+          description: the indicated configuration ACL once the posted changes have been applied
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/MethodConfigACL'
+        400:
+          description: Invalid content format
+        403:
+          description: User does not have correct permissions to manage namespace permissions or the indicated namespace was not found
+        500:
+          description: Internal Server Error
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+            - https://www.googleapis.com/auth/devstorage.full_control
```

Comment:
They are different endpoints in Agora - but I don't know how real that is. I can add a story to investigate and consolidate if possible.